src/sidebar/components/test/annotation-test.js

```diff
@@ -35,6 +35,21 @@ function annotationComponent() {
   });
 }
 
+/**
+ * Returns the controller for the action button with the given `label`.
+ *
+ * @param {Element} annotationEl - Annotation element
+ * @param {string} label - Button label
+ */
+function findActionButton(annotationEl, label) {
+  var btns = Array.from(annotationEl[0].querySelectorAll('annotation-action-button'));
+  var match = btns.find(function (btn) {
+    var ctrl = angular.element(btn).controller('annotationActionButton');
+    return ctrl.label === label;
+  });
+  return match ? angular.element(match).controller('annotationActionButton') : null;
```

Comment:
Yikes, it'd be great if there were a simpler way to find the flag button, but I guess there isn't?